### DISCUSSION TITLE
How to get file tts.wav file name

### DISCUSSION THREAD

**Discussion by Uzaker** [endink/Mediapipe4u-plugin#156] [Q&A]
When I set "save audio file" true , how can I get the name of the file?

**Comment by endink** [comment]
you can set `SaveAudioFilePath` property to control the file path.    
If `SaveAudioFilePath` is a relative file , it will be in <ProjectDir>/Saved/M4UAudio/<SaveAudioFilePath>.    
If `SaveAudioFilePath` is a absolute file path, it will in your path.

**Reply by Uzaker** [reply]
我能不能控制生成文件的名称？或者通过什么方式可以获取到这次生成的文件的名称？

**Reply by endink** [reply]
额，似乎没暴露这个 API，mark 一下，下个版本加上

**Comment by endink** [comment]
Mark, lets start support this

**Comment by endink** [comment]
fix in 20240520
